test/passing/tests/js_source.ml.ocp

```diff
@@ -9722,24 +9722,24 @@ type t =
   { field : ty
   (* Here is some verbatim formatted text:
      {v
-       starting at column 7
-     v}*)
+         starting at column 7
+         v}*)
```

Comment:
What is happening here ?

Reply:
We re-indent like this:

```diff
type t =
  { field : ty
      (* Here is some verbatim formatted text:
-    {v
-      starting at column 7
-    v}*)
+        {v
+        starting at column 7
+        v}*)
  }
```
which is consistent with the rest: aligned on the left.
The weird behavior is from `ocp-indent`.

To have a meaningful indentation, doc-comments should be used here.

Reply:
I see. It seems that ocp-indent indents differently `{v` blocks even in simple comments. Perhaps we should have something similar ?
The `Fmt_odoc` is not a good idea here because it might fail or escape characters in annoying ways.

Reply:
I added a special treatment for `{v v}`, indeed it was asked by Jane Street in #1132